lib/ansible/cli/galaxy.py

```diff
@@ -1003,10 +997,9 @@ def execute_install(self):
         force = context.CLIARGS['force'] or force_deps
 
         roles_left = []
-        if role_file:
-            if not (role_file.endswith('.yaml') or role_file.endswith('.yml')):
-                raise AnsibleError("Invalid role requirements file, it must end with a .yml or .yaml extension")
-
+        if role_file and not (role_file.endswith('.yaml') or role_file.endswith('.yml')):
```

Comment:
I dont know if this is actually easier to read, it doesnt seem to reduce lines of code or complexity